index.js

```diff
@@ -78,7 +58,8 @@ function msLogger (stream) {
 
 //opts must have appKey
 module.exports = function (opts) {
-
+  //this weird thing were some config is loaded first, then the rest later... not necessary.
```

Comment:
oh, I think it's refering to the way config is passed into secret-stack.
there are some options that are passed in when the SecretStack object is initialized.
https://github.com/ssbc/scuttlebot/blob/master/index.js#L165-L171
and then the rest is passed in when the sbot instance is created: https://github.com/ssbc/scuttlebot/blob/master/bin.js#L61

Really, it would be cleaner to just do the second one... but removing the first thing would break the tests which is why I havn't fixed it yet. (i think)